homeassistant/components/ultrasync/coordinator.py

```diff
@@ -0,0 +1,109 @@
+"""Provides the UltraSync DataUpdateCoordinator."""
+from datetime import timedelta
+import logging
+
+from async_timeout import timeout
+import ultrasync
+
+from homeassistant.const import CONF_HOST, CONF_PIN, CONF_SCAN_INTERVAL, CONF_USERNAME
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN, SENSOR_UPDATE_LISTENER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class UltraSyncDataUpdateCoordinator(DataUpdateCoordinator):
+    """Class to manage fetching UltraSync data."""
+
+    def __init__(self, hass: HomeAssistantType, *, config: dict, options: dict):
+        """Initialize global UltraSync data updater."""
+        self.hub = ultrasync.UltraSync(
+            user=config[CONF_USERNAME],
+            pin=config[CONF_PIN],
+            host=config[CONF_HOST],
+        )
+
+        self._init = False
```

Comment:
Is this used? I couldn't find the use-case, if not, please remove.

Reply:
:+1: Done